barcodes_generator_abstract/__manifest__.py

```diff
@@ -21,5 +21,5 @@
         "views/menu.xml",
     ],
     "demo": ["demo/res_users.xml"],
-    "external_dependencies": {"python": ["barcode"]},
+    "external_dependencies": {"python": ["python-barcode"]},
```

Comment:
Check if this change works in v16. If so (most likely case), apply this in that version. If no, integrate into the migration commit.